src/components/Cart/CartFooter.tsx

```diff
@@ -1,19 +1,35 @@
 import CartPrice from './CartPrice';
 import styled from '@emotion/styled';
-import { CartPriceVariant } from './CartPrice';
 
 interface CartFooterProps {
   price: number;
   shippingFee: number;
   totalPrice: number;
+  couponDiscount?: number;
 }
 
-function CartFooter({ price, shippingFee, totalPrice }: CartFooterProps) {
+function CartFooter({ price, shippingFee, totalPrice, couponDiscount = 0 }: CartFooterProps) {
+  const isZero = (value: number) => {
+    if (value === 0) {
+      return 0;
+    }
+
+    if (value > 0) {
+      return `-${couponDiscount}`;
+    }
+  };
```

Comment:
`isZero` 라는 이름만 보면 함수의 결과가 boolean을 반환한다고 생각했는데, discount를 반환하네요!?
이렇게 쓰이면 어떨까요?

```suggestion
  const adjustedDiscount = value !== 0 ? -couponDiscount : 0;
```

그냥 값으로 표현하는거죠..!

Reply:
함수명을 반환값의 타입과 부적절하게 지었네요..🥲 
분기 조건인 '0인지 아닌지'를 생각하다가 함수명에도 `is-`를 붙여버린 것 같아요.
피드백 주신 것처럼 로직을 값으로 변경했어요!

[refactor: 쿠폰 할인 금액의 음수 처리 로직 단순화](https://github.com/woowacourse/react-shopping-cart/pull/405/commits/2326f80ee4243f4a999278164fb9284ba5ba45dc)